### PR TITLE
Completed ParseFile and output display 

### LINKED ISSUE #5: Display event messages to screen
In preparation for processing server sent events from the web, let's read a stream of events from a text file, parse them then print them to the screen in a pleasing fashion.

### PR DESCRIPTION
**Pull request by 2dave**
Fixes #5

### CODE REVIEW

File: TD.SseParserTest/FileParserFixture.cs [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    public class FileParserFixture
+    {
+        [Fact]
+        public void BeginningFileParserTest()
+        {
+            var input = @".\test.txt";
+
+            using(FileStream fs = File.Create(input))
+            {
+                Byte[] info = new UTF8Encoding(true).GetBytes("data: foo\r\ndata: bar\r\n\r\nevent: next\r\n");
```
Comment (robmen) [inline TD.SseParserTest/FileParserFixture.cs]: `UTF8Encoding` is usually accessed by `Encoding.UTF8` static or something like that.

File: TwoDave.ServerSentEventsParser/Parser.cs [hunk 2]
```diff
@@ -58,5 +58,28 @@ public static SseMessage ParseMessage(string input, out string remainder)
 
             return message;
         }
+
+        public static SseMessage ParseFile(string path, out string remainder)
+        {
+            string fileline = null;
+            string filemessage = null;
+            
+            using(StreamReader file = new StreamReader(path))
```
Comment (robmen) [inline TwoDave.ServerSentEventsParser/Parser.cs]: How about `File.ReadAllLines()`?

File: TwoDave.ServerSentEventsParser/Program.cs [hunk 3]
```diff
@@ -7,16 +7,38 @@ class Program
...
-            Console.WriteLine("message.Data = {0}", message.Event);
-            Console.WriteLine("Remainder = {0}", remainder);
+                SseMessage message = new SseMessage();
+
+                message = Parser.ParseFile(path, out remainder);
+
+                Console.WriteLine("***Parsing file located at {0} ***", path);
+
+                // Could or should this be rewritten with either null conditionals or different form -> ? :
+                if (message.Id != null) //Is there a benefit to this? -> !string.IsNullOrEmpty(message.Id))
```
Comment (robmen) [inline TwoDave.ServerSentEventsParser/Program.cs:23]: This is fine. This avoids printing anything if null.

File: TwoDave.ServerSentEventsParser/Program.cs [hunk 4]
```diff
@@ -7,16 +7,38 @@ class Program
...
+            {
+                var remainder = "";
 
-            Console.WriteLine("message.Id = {0}", message.Id);
-            Console.WriteLine("message.Event = {0}", message.Data);
-            Console.WriteLine("message.Data = {0}", message.Event);
-            Console.WriteLine("Remainder = {0}", remainder);
+                SseMessage message = new SseMessage();
+
+                message = Parser.ParseFile(path, out remainder);
```
Comment (robmen) [inline TwoDave.ServerSentEventsParser/Program.cs:18]: I think next is to display all messages in the stream.

File: TD.SseParserTest/FileParserFixture.cs (hunk 1 above)
Comment (2dave) [reply TD.SseParserTest/FileParserFixture.cs]: Changed

File: TwoDave.ServerSentEventsParser/Program.cs (hunk 3 above)
Comment (2dave) [reply TwoDave.ServerSentEventsParser/Program.cs:23]: Got it

File: TwoDave.ServerSentEventsParser/Parser.cs (hunk 2 above)
Comment (2dave) [reply TwoDave.ServerSentEventsParser/Parser.cs]: Changed

File: TwoDave.ServerSentEventsParser/Program.cs (hunk 4 above)
Comment (2dave) [reply TwoDave.ServerSentEventsParser/Program.cs:18]: Ok. Will do next commit.

**Pull request merged**